docs/doc_extensions.py

```diff
@@ -6,11 +6,19 @@
 djangosetting
     Output an inline literal of the corresponding setting value. Useful for
     keeping documentation up to date without editing on settings changes.
+
+supportedversions
```

Comment:
I'd like to have a more descriptive name for this, like `dockerpyversions` or `buildpyversions` or `dockerimagepyversions` or similar.

`supportedversions` is too general.

Reply:
`buildpyversions` I like this one p: